client/tests/mock-inner-text.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Mock the `innerText` behaviour on HTML elements as JSDom does not
+ * include this full functionality by default. A simple implementation
+ * that allows the HTML content to be written and accessed.
+ */
+function mockInnerText() {
+  Object.defineProperty(HTMLElement.prototype, 'innerText', {
+    get() {
+      return this.textContent;
+    },
+    set(value) {
+      this.textContent = value;
+    },
```

Comment:
Curious: why not use `textContent` instead of `innerText` and skip the mocking then?

I know that they behave differently in regards to the getter, but I'm not so sure about the setter. This PR only uses the setter, so I'd assume we can use either one of them.

Is it just so that it's easier when we want to use `innerText` later?

Reply:
Good call, there was a different PR I recall reviewing where we needed this mock but ended up going a different direction. 

I agree that for this PR we could use inner intent instead. 

I'll make an update.

Reply:
Resolved